### PR TITLE
Windows CI: Port docker_cli_rm_test.go

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard jhoward@microsoft.com

This ports the tests in docker_cli_rm_test.go to run on Windows to Windows CI. Output from local run below - all tests in this file are ported and working :smiley: 

```
E:\go\src\github.com\docker\docker>runtest w TestRmContainer
Running test TestRmContainer
DOCKER_TEST_HOST=tcp://127.0.0.1:2375
DOCKER_HOST=tcp://127.0.0.1:2375
=== RUN   Test
INFO: Testing against a local daemon
PASS: docker_cli_rm_test.go:53: DockerSuite.TestRmContainerForceRemoveRunning   4.710s
PASS: docker_cli_rm_test.go:61: DockerSuite.TestRmContainerOrphaning    9.510s
PASS: docker_cli_rm_test.go:46: DockerSuite.TestRmContainerRunning      3.353s
PASS: docker_cli_rm_test.go:12: DockerSuite.TestRmContainerWithRemovedVolume    4.979s
PASS: docker_cli_rm_test.go:35: DockerSuite.TestRmContainerWithVolume   4.910s
OK: 5 passed
--- PASS: Test (30.35s)
PASS
ok      _/E_/go/src/github.com/docker/docker/integration-cli    30.559s

E:\go\src\github.com\docker\docker>
```

### CODE REVIEW

**Comment by jhowardmsft** [comment]
@icecrime @thaJeztah The failing win2lin test is known flakey.

**Comment by jhowardmsft** [comment]
All green. Submitted/merged a PR to stub out the flakey test.

File: integration-cli/docker_cli_rm_test.go [hunk 1]
```diff
@@ -9,34 +10,50 @@ import (
 )
 
 func (s *DockerSuite) TestRmContainerWithRemovedVolume(c *check.C) {
-	testRequires(c, DaemonIsLinux)
 	testRequires(c, SameHostDaemon)
 
-	dockerCmd(c, "run", "--name", "losemyvolumes", "-v", "/tmp/testing:/test", "busybox", "true")
+	prefix := ""
```
Comment (vdemeester) [inline integration-cli/docker_cli_rm_test.go]: @jhowardmsft we're doing this in multiple place, could we add a*util* function that returns `prefix` and `slash` according to the specified `daemonPlatform` (and thus making it easier to change, fix, enhance, maintain) ? :angel:

Comment (thaJeztah) [inline integration-cli/docker_cli_rm_test.go]: :+1:

Comment (thaJeztah) [inline integration-cli/docker_cli_rm_test.go]: I'm good with doing that in a separate PR, given that we already are using that code in several places

**Comment by thaJeztah** [comment]
LGTM

**Comment by vdemeester** [comment]
@thaJeztah SGTM, so LGTM 🐰

**Pull request merged**